From 9d62835c9a33e028b94f3dc6c6754310b28c1e4c Mon Sep 17 00:00:00 2001
From: xianbing Zhu <xianbing.zhu@linux.alibaba.com>
Date: Wed, 5 Jun 2024 16:59:18 +0800
Subject: [PATCH 072/129] net:stmmac: increase timeout for dma reset

Larger timeout for dma reset,to fix "Failed to reset the dma" issue,
which inspired from commit 9d14edfdeabf37d8d8f045e63e5873712aadcd6b:
           net: stmmac: increase the timeout for dma reset

Signed-off-by: xianbing Zhu <xianbing.zhu@linux.alibaba.com>
---
 drivers/net/ethernet/stmicro/stmmac/dwmac_lib.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/ethernet/stmicro/stmmac/dwmac_lib.c
+++ b/drivers/net/ethernet/stmicro/stmmac/dwmac_lib.c
@@ -24,7 +24,7 @@ int dwmac_dma_reset(void __iomem *ioaddr
 
 	return readl_poll_timeout(ioaddr + DMA_BUS_MODE, value,
 				 !(value & DMA_BUS_MODE_SFT_RESET),
-				 10000, 200000);
+				 10000, 1000000);
 }
 
 /* CSR1 enables the transmit DMA to check for new descriptor */
